db/migrate/20161122010755_fixes_paid_count_function.rb

```diff
@@ -0,0 +1,23 @@
+class FixesPaidCountFunction < ActiveRecord::Migration
+  def up
+    execute <<-SQL
+    CREATE OR REPLACE FUNCTION paid_count(rewards) RETURNS bigint AS $$
+      SELECT count(*)
+      FROM payments p
+      JOIN contributions c ON c.id = p.contribution_id
+      JOIN projects pr ON c.project_id = pr.id
+      WHERE p.state = (CASE WHEN pr.state = 'failed' THEN 'refunded' ELSE 'paid' END)
```

Comment:
@vicnicius lets use when failed pr.state in ('refunded', 'pending_refund', 'paid')